config/config.go

```diff
@@ -24,7 +25,10 @@ func LoadFile(filename string) (*Config, error) {
 
 var (
 	DefaultModule = Module{
-		Version: 2,
+		Version:        2,
+		Retries:        3,
+		MaxRepititions: 25,
+		Timeout:        60,
```

Comment:
Just to clarify, if you specify the value as a full `time.Duration` and not a multiplier value for `time.Second` then the value in the `snmp.yaml` file will have to be specified as the following to be 1 second:

```yaml
default:
  timeout: 1000000000
...
```

In my opinion, this would be very annoying to the end user. Do you want the configuration to work that way or for the value to be specified in the `snmp.yml` file in terms of a multiplier of a larger increment like `time.Second` or `time.Millisecond`? 

If you are confused by my (mis)use of the `time.Duration` type I can change the type in `config.Module` to be `int64` and then do appropriate run-time casting. In my mind doing a cast from `int64` to `time.Duration` is extra work that the yaml tool is taking care of for me. Though I would agree that it could add clarity to the code to do so.